ios/Capacitor/Capacitor/CapacitorBridge.swift

```diff
@@ -95,7 +95,7 @@ open class CapacitorBridge: NSObject, CAPBridgeProtocol {
     public static let capacitorSite = "https://capacitorjs.com/"
     public static let fileStartIdentifier = "/_capacitor_file_"
     public static let httpInterceptorStartIdentifier = "/_capacitor_http_interceptor_"
-    public static let httpsInterceptorStartIdentifier = "/_capacitor_https_interceptor_"
```

Comment:
Since it's public, removing it would be a breaking change, so better mark it as deprecated and we will remove it in next major.

Reply:
Done via: ac97cd4